depends/common/libpng/flags.txt

```diff
@@ -1 +1 @@
--DBUILD_SHARED_LIBS=0 -DCMAKE_POSITION_INDEPENDENT_CODE=0 -DPNG_SHARED=0
+-DBUILD_SHARED_LIBS=0 -DCMAKE_POSITION_INDEPENDENT_CODE=0 -DPNG_SHARED=0 -DPNG_ARM_NEON=off
```

Comment:
The text `Explicitly enable Neon for arm so it works for iOS/tvOS` not match direct the done define where is `disable` :smile:

Reply:
Whoops, so I never committed `on` so I never even tested it :(